Shared/Samples/Control annotation sublayer visibility/ControlAnnotationSublayerVisibilityView.Model.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension ControlAnnotationSublayerVisibilityView {
+    /// The view model for this sample.
+    @MainActor
```

Comment:
It seems the only main actor isolation required method in this data model is `loadMobileMapPackage()`, so I think probably just annotate the method itself. Up to you.